.gitignore

```diff
@@ -0,0 +1,2 @@
+
+tags
```

Comment:
Is there a reason for this gitignore?

Reply:
ctags, it's just to make easier, otherwise I have to stash my ctags file every time before push

Reply:
You can commit specific files by doing `git commit lua/` though no?